maintainers/maintainer-list.nix

```diff
@@ -4519,6 +4519,12 @@
       fingerprint = "0200 3EF8 8D2B CF2D 8F00  FFDC BBB3 E40E 5379 7FD9";
     }];
   };
+  GaetanLepage = {
+    email = "gaetan@glepage.com";
+    github = "GaetanLepage";
+    githubId = 33058747;
+    name = "Gaetan Lepage";
+  };
```

Comment:
This should be in a separate commit named: `maintainers: add GaetanLepage`

Reply:
How exactly should I address this ? Should I revert this change and make a new separate commit ? Wouldn't this pollute the history tree ?

Reply:
You should only have two commits, one for maintainer and the other for adding the package. To do so you can either use git rebase or git reset, make new commits and force push them.